src/OnboardingSPA/steps/SiteGen/Editor/index.js

```diff
@@ -81,9 +81,16 @@ const StepSiteGenEditor = () => {
 				populateFontsInPreviewSettings( newPreviewSettings );
 		}
 
+		let blockGrammar = '';
+		[ 'header', 'content', 'footer' ].forEach( ( part ) => {
```

Comment:
I see this array is used in a lot of places, should we move this to a constant or a util func that combines and gives the Block Grammar?